### ISSUE TITLE
documentation re-alignment

### ISSUE THREAD

**Issue by shoebsd31** [hyperledger/indy-sdk#2376]
Currently in the [documentation](https://github.com/hyperledger/indy-sdk/#ubuntu-based-distributions-ubuntu-1604-and-1804), it would be great if it specified the keys that needs to replaced first before the command statements. For example if we could rephrase this as follows :

**Current Implementation**
### Ubuntu based distributions (Ubuntu 16.04 and 18.04)
It is recommended to install the SDK packages with APT:

    sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CE7709D068DB5E88
    sudo add-apt-repository "deb https://repo.sovrin.org/sdk/deb (xenial|bionic) {release channel}"
    sudo apt-get update
    sudo apt-get install -y {library}

* {library} must be replaced with libindy, libnullpay, libvcx or indy-cli.
* (xenial|bionic) xenial for 16.04 Ubuntu and bionic for 18.04 Ubuntu.
* {release channel} must be replaced with master, rc or stable to define corresponded release channel.
Please See the section "Release channels" above for more details.

**Proposed Implementation :** 
### Ubuntu based distributions (Ubuntu 16.04 and 18.04)
It is recommended to install the SDK packages with APT:
Please replace the literals before executing the apt statements :
* {library} must be replaced with libindy, libnullpay, libvcx or indy-cli.
* (xenial|bionic) xenial for 16.04 Ubuntu and bionic for 18.04 Ubuntu.
* {release channel} must be replaced with master, rc or stable to define corresponded release channel.
Please See the section "Release channels" above for more details.

    sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CE7709D068DB5E88
    sudo add-apt-repository "deb https://repo.sovrin.org/sdk/deb (xenial|bionic) {release channel}"
    sudo apt-get update
    sudo apt-get install -y {library}

**Comment by ianco** [comment]
@shoebsd31  good catch on the docs, do you want to do a PR to do the update?

**Comment by shoebsd31** [comment]
> @shoebsd31  good catch on the docs, do you want to do a PR to do the update?
> 

OK I will update it

**Comment by ianco** [comment]
> > @shoebsd31  good catch on the docs, do you want to do a PR to do the update?
> 
> OK I will update it

Awesome I'll look forward to seeing your PR!  Remember to DCO sign your commits (`git commit --signoff` or `git commit -s`)

**Comment by shoebsd31** [comment]
> > > @shoebsd31  good catch on the docs, do you want to do a PR to do the update?
> > 
> > 
> > OK I will update it
> 
> Awesome I'll look forward to seeing your PR! Remember to DCO sign your commits (`git commit --signoff` or `git commit -s`)

I have raised a PR for this issue :)
